README.md

```diff
@@ -4,7 +4,14 @@
 ![Lint Code Base](https://github.com/sleighzy/ansible-zookeeper/workflows/Lint%20Code%20Base/badge.svg)
 ![Ansible Lint](https://github.com/sleighzy/ansible-zookeeper/workflows/Ansible%20Lint/badge.svg)
 
-Ansible role for installing and configuring Apache ZooKeeper on RedHat 7 and 8.
+Ansible role for installing and configuring Apache ZooKeeper
+
+| Platform / Status | Supported           | Tested             |
```

Comment:
Fine for now. I have some additional test framework changes coming so can I tweak once those go in.